router.py

```diff
@@ -216,6 +216,9 @@ async def _update_platform(self, api_callable: Callable):
                     "The last requested resulted in a token error - so renewing token"
                 )
                 await self.renew_token()
+            if self._connect_error:
```

Comment:
Why was this necessary? Did you find non-token connection errors that resolved with token renewal?

Reply:
Yes, after rebooting the router this integration would not reconnect. After a reboot a `connect_error` occurs instead of a `token_error`.